tests/bootstrap.php

```diff
@@ -44,6 +44,8 @@ function xwp_filter_active_plugins_for_phpunit( $active_plugins ) {
 	function() {
 		// Manually load the plugin.
 		require dirname( __DIR__ ) . '/stream.php';
+		$plugin = $GLOBALS['wp_stream'];
```

Comment:
The concept looks good! Could we use `wp_stream_get_instance()` which is our official public API for getting the plugin instnace?